modules/sr/robot/compass.py

```diff
@@ -16,4 +16,4 @@ def get_heading(self) -> float:
         """
         x, _, z = self._compass.getValues()
         heading = atan2(x, z) % tau
-        return add_jitter(heading, 0, tau)
+        return add_independent_jitter(heading, 0, tau, 0.4, True)
```

Comment:
Suggest passing `0.4` and `True` as kwargs for clarity.
```suggestion
        return add_independent_jitter(heading, 0, tau, std_dev_percent=0.4, can_wrap=True)
```